snmp_cisco/metadata.csv

```diff
@@ -0,0 +1,179 @@
+snmp.bgpPeerHoldTime,gauge,,second,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The time interval in seconds for the Hold Timer established with the peer.,0,cisco,
+snmp.bgpPeerKeepAlive,gauge,,second,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The time interval in seconds for the KeepAlive timer established with the peer.,0,cisco,
+snmp.bgpPeerHoldTimeConfigured,gauge,,second,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The time interval in seconds for the Hold Time configured for this BGP speaker with this peer.,0,cisco,
+snmp.bgpPeerKeepAliveConfigured,gauge,,second,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The time interval in seconds for the KeepAlive timer configured for this BGP speaker with this peer.,0,cisco,
+snmp.bgpPeerMinASOriginationInterval,gauge,,second,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The time interval in seconds for the MinASOriginationInterval timer.,0,cisco,
+snmp.bgpPeerInUpdates,rate,,message,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The number of BGP UPDATE messages received on this connection.,0,cisco,
+snmp.bgpPeerOutUpdates,rate,,message,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The number of BGP UPDATE messages transmitted on this connection.,0,cisco,
+snmp.bgpPeerInTotalMessages,rate,,message,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The total number of messages received from the remote peer on this connection.,0,cisco,
+snmp.bgpPeerOutTotalMessages,rate,,message,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The total number of messages transmitted o the remote peer on this connection.,0,cisco,
+snmp.bgpPeerFsmEstablishedTransitions,rate,,,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] The total number of times the BGP FSM transitioned into the established state.,0,cisco,
+snmp.bgpPeerFsmEstablishedTime,gauge,,second,,[Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] [Cisco CSR1000v] This timer indicates how long (in seconds) this peer has been in the Established state or how long since this peer was last in the Established state.,0,cisco,
+snmp.ifSpeed,gauge,,,,"An estimate of the interface's current bandwidth in bits per second, or the nominal bandwidth.",0,cisco,
+snmp.ifHighSpeed,gauge,,,,"An estimate of the interface's current bandwidth in units of 1,000,000 bits per second, or the nominal bandwidth.",0,cisco,
+snmp.ifBandwidthInUsage.rate,gauge,,percent,,"The percent rate of used received bandwidth.",0,cisco,
+snmp.ifBandwidthOutUsage.rate,gauge,,percent,,"The percent rate of used sent bandwidth.",0,cisco,
+snmp.rttMonLatestRttOperSense,gauge,,,,A sense code for the completion status of the latest RTT operation.,0,cisco,
+snmp.rttMonCtrlOperTimeoutOccurred,gauge,,,,This object will change its value for all RttMonRttTypes.,0,cisco,
```

Comment:
All those metrics will now be duplicated in `snmp_cisco` and in `snmp` integration ? Will lead to maintenance burden when we need to add/update/delete metrics. I think we need to address that if we plan to create dozen of new tiles.

cc @gmarabout

Reply:
As far as I understood, we use `snmp_cisco` just to show to customers that we support `cisco` devices. As there is no check involved in this integrations, metrics should still live in the `snmp`, as per profile. If guess that if we were to move `cisco` metrics to `snmp_cisco` we should also change the check code to take into account that some metrics the `check` in `snmp` is collecting will be declared in a separate folder. 

Here I'm not sure if metrics declaration, beside showing values in the tile, are used to send metrics from the agent to the app, or if they are used to accept metrics on the app side.

In other words, if we have `snmp` declaring `snmp.some_snmp_metric` and `snmp_check` declaring `snmp.some_snmp_cisco_metric`, how will the customer need to setup  their integrations to make sure they receive both, knowing that the `check` is only on `snmp` side? Will they need to install both?